### ISSUE TITLE
Support go jsonnet too?

### ISSUE THREAD

**Issue by ghostsquad** [Banno/asdf-jsonnet#1]
Would you be interested in supporting go-jsonnet?

**Comment by craigfurman** [comment]
I wrote an asdf plugin for go-jsonnet a while back: https://gitlab.com/craigfurman/asdf-go-jsonnet

Now that I think about it, one plugin could represent both implementations of jsonnet using different version strings (0.17.0 and go-0.17.0, for example). This one is already in https://github.com/asdf-vm/asdf-plugins, so I guess it makes sense to do it here. I'd be happy to PR go-jsonnet support if the maintainers are interested?

**Comment by ghostsquad** [comment]
@craigfurman yep I use your plugin currently. It would be very nice to ensure a specific implementation is used. I've made some contributions to the Go version that makes it much faster in some cases that the original.

**Comment by craigfurman** [comment]
That's great, I'm a big fan of go-jsonnet's speed. @nikolay you've got every commit in this repo, but I see it's been a while. Are you interested in maintaining support for go-jsonnet versions in this repo? If not, that's fine, and I'll likely look into PR'ing https://gitlab.com/craigfurman/asdf-go-jsonnet into https://github.com/asdf-vm/asdf-plugins.

**Comment by nikolay** [comment]
@craigfurman Yeah, it's been a while. And a while since I last used Jsonnet. Are they now equivalent as back then they weren't?

**Comment by ghostsquad** [comment]
Go-jsonnet is the future, even the maintainers want to encourage more adoption of it over the C++ version.
